Library/Formula/geocouch.rb

```diff
@@ -18,11 +18,11 @@ class Geocouch < Formula
   depends_on "erlang" => :build
 
   def couchdb_share
-    HOMEBREW_PREFIX/"share/couchdb"
+    HOMEBREW_PREFIX/pkgshare
   end
 
   def geocouch_share
```

Comment:
Pretty sure this function can be removed and all callers replaced with `pkgshare`